packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -19,18 +19,25 @@ import { CommandContribution, CommandRegistry, Command } from '@theia/core';
 import { CommandService } from '@theia/core/lib/common/command';
 import TheiaURI from '@theia/core/lib/common/uri';
 import URI from 'vscode-uri';
+import { ContextKeyService } from '@theia/plugin-ext/lib/main/browser/context-key/context-key';
```

Comment:
I think the error from travis may have to do with the import statement.
I believe it should be 
```ts
import { ContextKeyService } from '@theia/core/lib/browser/context-key-service';
```

Reply:
oh wait I see now you're trying to refer to the one found in the `plugin-ext` extension.